docs/modular-resources/upload/_index.md

```diff
@@ -30,21 +30,37 @@ To upload your custom module to the [Viam registry](https://app.viam.com/registr
 
 1. First, [install the Viam CLI](/manage/cli/#install) and [authenticate](/manage/cli/#authenticate) to Viam, from the same machine that you intend to upload your module from.
 
-1. Next, run the `viam module create` command to select a new custom module name and generate module metadata.
+2. Next, run the `viam module create` command to select a new custom module name and generate module metadata.
+   Run this command according to your desired visibility for your module:
```

Comment:
```suggestion
   Run this command according to your desired visibility for your module, which is by default private at creation:
```

Reply:
@andf-viam This is the best I can think of and sounds super awkward. Any ideas to make it fit in better with the flow here?

Reply:
It just seems a little confusing to mention it here-- I'm like "run this command for what you want" and then like "actually that doesn't matter because it's created as private then you edit the meta.json file to set visibility"